Get the pixel area from the photom reference file

philhodge opened the pull request:
For all instruments other than NIRSpec, the pixel area (solid angle) is not gotten from the photom reference file instead of from the area reference file.

philhodge commented:
The IFU cube det_image_ch1-short_s3d.fits and also det_image_ch1-short_s3d.fits were made using version 0.14.0 of JWST.  A number of significant changes have been made to the pipeline since then, including changes to the units of an IFU cube (now MJy / sr).  You should do the test again using JWST 0.14.3, i.e. the latest develop version.  I would suggest starting with the raw file and running calwebb_detector1.py and calwebb_spec2.py.

hbushouse commented:
Where did these comments come from? I don’t see them anywhere within #4270. Is this something really old that github just finally decided to pass along?

From: Phil Hodge <notifications@github.com>
Reply-To: spacetelescope/jwst <reply@reply.github.com>
Date: Monday, February 17, 2020 at 2:51 PM
To: spacetelescope/jwst <jwst@noreply.github.com>
Cc: Howard Bushouse <bushouse@stsci.edu>, Review requested <review_requested@noreply.github.com>
Subject: Re: [spacetelescope/jwst] Get the pixel area from the photom reference file (#4270)


The IFU cube det_image_ch1-short_s3d.fits and also det_image_ch1-short_s3d.fits were made using version 0.14.0 of JWST. A number of significant changes have been made to the pipeline since then, including changes to the units of an IFU cube (now MJy / sr). You should do the test again using JWST 0.14.3, i.e. the latest develop version. I would suggest starting with the raw file and running calwebb_detector1.py and calwebb_spec2.py.

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub<https://github.com/spacetelescope/jwst/pull/4270?email_source=notifications&email_token=ADCDVNA7DAUZI6BC57IZSKDRDLTDFA5CNFSM4JN4W6LKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEL7P4QA#issuecomment-587136576>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADCDVNFJHWYRYCQO2Z5Z3W3RDLTDFANCNFSM4JN4W6LA>.

philhodge commented:
Sorry, that was a dumb mistake!  I had a Jira ticket open in one tab, and I was looking at an old GitHub PR (this one) in another tab.  I intended to add a comment to the Jira ticket, but I added it to this PR instead.  When I realized my mistake, I deleted the comment from this PR (that's why you don't see it here), and I added it to the Jira ticket.
